mainapp/templates/mainapp/user.html

```diff
@@ -28,27 +28,36 @@
 {% endblock %}
 {% block body %}
 <div class="container">
-	<div class="col-12">
-		<h1>{{ owner.profile.display_name }}'s user page</h1>
+	<div class="row">
+	<div class="col-lg-8">
+		{% if owner.avatar %}
+		<div class="col-md-2">
+			<h1><img src="{{ owner.avatar }}" alt=""></h1>
+		</div>
+		{% endif %}
+		<div class="col-md-8">
+			<h1><a target="_blank" href="https://www.openstreetmap.org/user/{{ owner.display_name }}/">{{ owner.display_name }}</a></h1>
```

Comment:
I'd remove the final slash, i.e. use `https://www.openstreetmap.org/user/{{ owner.display_name }}`.